docs/source/markdown/options/hooks-dir.md

```diff
@@ -0,0 +1,15 @@
+####> This option file is used in:
+####>   podman build, farm build
+####> If file is edited, make sure the changes
+####> are applicable to all of those.
+#### **--hooks-dir**=*path*
+
+Each *.json file in the path configures a hook for buildah build containers. For more details on the syntax of the JSON files and the semantics of hook injection. Buildah currently support both the 1.0.0 and 0.1.0 hook schemas, although the 0.1.0 schema is deprecated.
+
+This option may be set multiple times; paths from later options have higher precedence.
+
+For the annotation conditions, buildah uses any annotations set in the generated OCI configuration.
```

Comment:
```suggestion
For the annotation conditions, Buildah uses any annotations set in the generated OCI configuration.
```

Reply:
This is a backport so we can't really fix it here, need to do it on main